src/components/Docs/Docs.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text } from '@chakra-ui/react';
+
+export function DocsURL({ url, children }: { url: string | undefined, children: React.ReactNode; }) {
```

Comment:
Is there a use case you were thinking of needing children over a simple text string param that is required?

Reply:
no, refactored